src/wp-includes/class-wp-text-diff-renderer-table.php

```diff
@@ -515,12 +525,22 @@ public function __get( $name ) {
 	 *
 	 * @param string $name  Property to check if set.
 	 * @param mixed  $value Property value.
-	 * @return mixed Newly-set property.
 	 */
 	public function __set( $name, $value ) {
 		if ( in_array( $name, $this->compat_fields, true ) ) {
-			return $this->$name = $value;
+			$this->$name = $value;
+			return;
 		}
+
+		_doing_it_wrong(
+			__METHOD__,
+			sprintf(
+			// translators: 1: The name of the non-existent class property.
+				__( 'The "%1$s" property isn\'t defined. Dynamic properties are deprecated in PHP 8.2 and above.' ),
+				$name
+			),
+			'6.3.0'
```

Comment:
Use 6.2.0

Reply:
Good catch.
Fixed in a9e364eaf792b85ba0bb611f6642e40e033b9a26